enhancements/update/overriding-blocked-edges/flow.dot

```diff
@@ -0,0 +1,25 @@
+digraph updateDecisions {
+	new [ label="New edge available" ];
+	test [ label="Do local tests pass?" ];
```

Comment:
I've pushed:

```console
$ git --no-pager diff 0d8c588..f606f41 -- enhancements/update/overriding-blocked-edges/
diff --git a/enhancements/update/overriding-blocked-edges/README.md b/enhancements/update/overriding-blocked-edges/README.md
index b011337..775ef0b 100644
--- a/enhancements/update/overriding-blocked-edges/README.md
+++ b/enhancements/update/overriding-blocked-edges/README.md
@@ -38,11 +38,12 @@ Between those two, and similar efforts, I don't think that broadcasting edge-pul
 Because do we support them if they hit some new bug?
 Our docs around this are [currently wiggly][off-graph-support], but I expect there will be a lot of pain if we offer blanket support for folks taking any blocked edge we have ever served in a fast/stable channel.
 
-Also note that the flow is always going through "Do local tests pass?".
+Also note that the flow is always going through "Are there failing local tests?".
 We want users testing in candidate and fast to help turn up issues with their particular cluster flavor that we might not be catching in CI.
 We also want them reporting Telemetry/Insights when possible too, so we can pick out lower-signal feedback that they might miss.
 Or at least filing bugs if they find an issue and aren't reporting it via Telemetry/Insights.
 Phased roll-outs ([when we get them][phased-rollouts]) allow us to minimize the impact of "nobody explicitly tested this flavor/workflow (enough times, for flaky failures)", but is ideally not the first line of defense.
+But having no local tests and relying on the rest of the fleet reporting Telemetry/Insights is still supported, so users could work through the flow with "We have no failing local tests, because we don't run local tests" if they want.
 
 If I was administering a production cluster, I'd personally be conservative and avoid both dashed and orange "yes" paths.
 
diff --git a/enhancements/update/overriding-blocked-edges/flow.dot b/enhancements/update/overriding-blocked-edges/flow.dot
index bd27504..42c8926 100644
--- a/enhancements/update/overriding-blocked-edges/flow.dot
+++ b/enhancements/update/overriding-blocked-edges/flow.dot
@@ -1,6 +1,6 @@
 digraph updateDecisions {
        new [ label="New edge available" ];
-       test [ label="Do local tests pass?" ];
+       test [ label="Are there failing local tests?" ];
        pulled [ label="Does Red Hat pull the edge?" ];
        localConfidence [ label="Are we confident in\nour local test coverage?" ];
        redHatConfidence [ label="Are we confident Red Hat's\nreasoning is exhaustive?" ];
@@ -10,9 +10,9 @@ digraph updateDecisions {
        wait [ label="Wait for a new edge" ];
 
        new -> test;
-       test -> wait [ label="no" ];
+       test -> wait [ label="yes" ];
        wait -> new;
-       test -> pulled [ label="yes" ];
+       test -> pulled [ label="no" ];
        pulled -> update [ label="no" ];
        pulled -> localConfidence [ label="yes" ];
        localConfidence -> update [ label="yes"; color="orange"; style="dashed" ];
diff --git a/enhancements/update/overriding-blocked-edges/flow.svg b/enhancements/update/overriding-blocked-edges/flow.svg
...
```

to try to address this use-case.

Reply:
> But having no local tests and relying on the rest of the fleet reporting Telemetry/Insights is still supported, so users could work through the flow with "We have no failing local tests, because we don't run local tests" if they want.

I can live with the current flow, but I would have made it explicit.